sidebars.json

```diff
@@ -169,7 +169,7 @@
       },
       "items": [
         {
-          "id": "development_suite/company/create",
+          "id": "development_suite/company/create-company",
```

Comment:
Are we sure about this path change? If yes we have to add the 310redirect from the old path to the new one

Reply:
I believe having the file named create-company would be helpful, as I had trouble finding it myself. I renamed it and updated the ID for consistency. Which redirect should we add? I have already made changes to the links that were causing issues 🤔

Reply:
@malta895 can you explain to Rick what to do for the redirect?